src/coordinatorapi.cpp

```diff
@@ -544,6 +548,17 @@ DmtcpMessage CoordinatorAPI::sendRecvHandshake(DmtcpMessage msg,
       .Text("Connection rejected by the coordinator.\n"
             " Reason: This process has a different computation group.");
   }
+  // Coordinator also prints this, but its stderr may go to /dev/null
+  if (msg.type == DMT_REJECT_NOT_RESTARTING) {
+    const char *coordinatorHost;
+    int coordinatorPort;
+    Util::getCoordHostAndPort(COORD_ANY, &coordinatorHost, &coordinatorPort);
+    JNOTE ("\n\n*** Computation not in RESTARTING or CHECKPOINTED state."
+        "\n***Reject incoming computation process requesting restart."
+        "\n***Could the coordinator be still running a previous computation?"
```

Comment:
DMT_REJECT_NOT_RESTARTING does mean that the coordinator is running a different computation.  Let's also remove " or CHECKPOINTED" from the message since this is misleading.